Formula/pv.rb

```diff
@@ -53,3 +53,15 @@ def install
  #ifdef __cplusplus
  extern "C" {
  #endif
+diff --git a/src/pv/number.c b/src/pv/number.c
```

Comment:
Has this new patch been upstreamed?

Reply:
No, that is added by mine, and not report to upstream yet.

Reply:
OK, now I report this to `pv` via [Contact](http://www.ivarch.com/personal/contact.shtml). What should I do next?